src/test/rustdoc-gui/sidebar-mobile.goml

```diff
@@ -29,3 +29,9 @@ assert-css: (".sidebar", {"display": "block", "left": "-1000px"})
 
 // Check that the topbar is visible
 assert-property: (".mobile-topbar", {"clientHeight": "45"})
+
+// Check that clicking an element from the sidebar scrolls to the right place
+// so the target is not obscured by the topbar.
+click: ".sidebar-menu-toggle"
+click: ".sidebar-links a"
+assert-position: ("#method\.must_use", {"y": 45})
```

Comment:
If I understand correctly, you're just checking the position of this item, not the scroll position. To check the scroll position, you can instead check the property `scrollTop` on the `<body>` element.

PS: please keep this check though, it'll allow us to have a "reference" point. ;)

Reply:
This is actually the property I want to check - what is the position of `#method.must_use` _relative to the viewport_. That should be 45, so long as the topbar is 45 pixels wide.

I _don't_ want to check the body's scroll position, because that will change anytime the layout of the page changes, resulting in a brittle test.

Reply:
Oh I see, thanks for the explanations! Then all good for me!